omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PersonAttributeResource1_8.java

```diff
@@ -76,32 +77,12 @@ public DelegatingResourceDescription getRepresentationDescription(Representation
 	@Override
 	public DelegatingResourceDescription getCreatableProperties() {
 		DelegatingResourceDescription description = new DelegatingResourceDescription();
-		description.addProperty("value");
 		description.addRequiredProperty("attributeType");
+		description.addProperty("value");
 		description.addProperty("hydratedObject");
 		return description;
 	}
 
-    /*
-        We may need to hydrate Attributables in the request. To do that we need to know
-        its type. So we can't set hydratedObject until attributeType has been set.
-        Since SimpleObject is a LinkedHashMap, it is ordered, and the following method is
-        overridden to ensure hydratedObject will appear last in the properties.
-     */
-    @Override
-    public Object create(String parentUniqueId, SimpleObject post, RequestContext context) throws ResponseException {
-        uglyMethodToEnsureHydratedObjectWillBeSetLast(post);

```

Comment:
yes it is ensured by the order properties are added in getCreatableProperties() method. And since   [DelegatingResourceDescription](https://github.com/openmrs/openmrs-module-webservices.rest/blob/master/omod-common/src/main/java/org/openmrs/module/webservices/rest/web/resource/impl/DelegatingResourceDescription.java#L33-L35) uses LinkedHashMap internally the order is guaranteed to be maintained.